source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -155,7 +155,13 @@ You’re now about to finish configuring SAML for Mattermost!
 Configure SAML Sign-in for Mattermost
 --------------------------------------
 
-1. Start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
+Create a metadata URL by appending "FederationMetadata/2007-06/FederationMetadata.xml" to the root URL of the ADFS server, for example: ``https://<adfs.domain.com>/federationmetadata/2007-06/FederationMetadata.xml>``. 
+
+Next, start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**, paste metadata URL in the **Identity Provider Metadata URL** field, and then select **Get SAML Metadata from IdP**.
+
+This populates the **SAML SSO URL** and the **Identity Provider Issuer URL** fields automatically and the Identity Provider Public Certificate is also downloaded from the server and set locally.
+
+Alternatively you can enter the following fields manually:
```

Comment:
Only the last three fields (from the 6 enumerated) are being impacted by the user entering the metadata Url). The first three can be selected for both alternatives. So I would move the "Alternatively..." statement on top of the last three fields (SSO, IdpUrl and cert) only.

Reply:
Thanks! Updated it.